docs/getting-started/installation.md

```diff
@@ -15,8 +15,9 @@ In this section you will find an aggregation of the different ways to install Tr
     [trivy]
     name=Trivy repository
     baseurl=https://aquasecurity.github.io/trivy-repo/rpm/releases/$RELEASE_VERSION/\$basearch/
-    gpgcheck=0
+    gpgcheck=1
     enabled=1
+    gpgkey=https://aquasecurity.github.io/trivy-repo/deb/public.key
```

Comment:
Should we copy the key under `trivy-repo/rpm` as it is currently located under `trivy-repo/deb`.

Reply:
Merged. Thanks.